frontend/packages/integration-tests-cypress/views/list-page.ts

```diff
@@ -35,6 +35,19 @@ export const listPage = {
         cy.byLegacyTestID('dropdown-button').click();
       });
     },
+    clickFilterDropdown: () => {
+      cy.get('.pf-c-toolbar__content-section').within(() => {
+        cy.get('button.pf-c-select__toggle').click();
+      });
+    },
+    clearAllFilters: () => {
+      cy.log('Clearing all filters');
+      cy.get('.pf-c-toolbar__item').within(() => {
+        cy.get('button')
+          .last()
```

Comment:
I prefer a more deterministic selection, how about `cy.contains('Clear all filters').click()`?

Reply:
that's it, much simpler!

Reply:
`cy.contains('Clear all filters').click()` doesn't seem working since the text was already pesudo translated when we visited the URL

Reply:
ah..ok, makes sense :-)  
However, the selector `cy.get('.pf-c-toolbar__item').within...` isn't quite correct.  There are many `.pf-c-toolbar__item`'s in the toolbar (filter dropdown, name input, chips, etc..).  I think what you want is:
`cy.get('.pf-c-toolbar__content').within...`.

Reply:
Ok, I got you ~~ will update